### PR TITLE
adapt new variable names and cohorts

### LINKED ISSUE #42: Fixes to variable names for panc-prostate-bladder bundle
- [x] Update '{SP}-sor' to '{SP}_sor' to reflect change in [GENIE BPC No PHI Data Elements Catalog](https://www.synapse.org/#!Synapse:syn21431364)
- [x] Update 'tt_first_index_ca' variable name to 'dob_first_index_ca' to reflect change in BPC variables
- [x] Update bpc to cbioportal mapping table version for three cohorts

### PR DESCRIPTION
**Pull request by hhunterzinck**
Fixes #42

### CODE REVIEW

File: geniesp/bpc_config.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ class Bladder(BpcProjectRunner):
     # Sponsorted project name
     _SPONSORED_PROJECT = "BLADDER"
     # Redcap codes to cbioportal mapping synid and form key is in
-    _REDCAP_TO_CBIOMAPPING_SYNID = "syn25712693.14"
+    _REDCAP_TO_CBIOMAPPING_SYNID = "syn25712693.21"
```
Comment (thomasyu888) [inline geniesp/bpc_config.py:115]: Why 21 vs 20?

**Review by thomasyu888 — APPROVED** [review]

File: geniesp/bpc_redcap_export_mapping.py [hunk 2]
```diff
@@ -844,7 +844,7 @@ def run(self):
         # Must use this to determine release scope!
         data_elements = self.syn.tableQuery(
             f"select distinct variable from {self._DATA_ELEMENT_SYN_ID} "
-            f'where "{self._SPONSORED_PROJECT}-sor" in '
+            f'where "{self._SPONSORED_PROJECT}_sor" in '
```
Comment (thomasyu888) [inline geniesp/bpc_redcap_export_mapping.py:847]: Why the change now?

Comment (hhunterzinck) [reply geniesp/bpc_redcap_export_mapping.py:847]: @xindiguo updated in source table.   I think the hyphen was causing issues with table queries with other scripts for some reason.

File: geniesp/bpc_config.py (hunk 1 above)
Comment (hhunterzinck) [reply geniesp/bpc_config.py:115]: I had to manually change the flags for only the bladder cohort to include some of the radiation therapy variables.  panc and prostate could be 21 as well.
